projects/aca-content/src/lib/components/sidenav/user-menu/user-menu.component.ts

```diff
@@ -50,9 +50,26 @@ export class UserMenuComponent implements OnInit {
   @Input()
   data: { items: any[] };
 
+  @ViewChild(MatMenu)
+  menu: MatMenu;
+
+  @ViewChildren(ToolbarMenuItemComponent)
+  toolbarMenuItems: QueryList<ToolbarMenuItemComponent>;
+
   ngOnInit() {
     if (this.data?.items) {
       this.data.items.sort((a, b) => a.order - b.order);
     }
   }
+
+  ngAfterViewInit() {
+    const menuItems: MatMenuItem[] = this.toolbarMenuItems
```

Comment:
no need to set type for menuItems (at left side) because it's set automatically based on filter function

Reply:
removed